### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by ericgaspar**
- Upgrade to version 3.4.0

### CODE REVIEW

**Comment by yalh76** [comment]
!testme

File: scripts/restore [hunk 1]
```diff
@@ -42,8 +42,6 @@ datadir=$(ynh_app_setting_get --app=$app --key=datadir)
 #=================================================
 ynh_script_progression --message="Validating restoration parameters..."
 
-ynh_webpath_available --domain=$domain --path_url=$path_url \
-	|| ynh_die --message="Path not available: ${domain}${path_url}"
```
Comment (fflorent) [inline scripts/restore:46]: Why this removal?

Comment (ericgaspar) [reply scripts/restore:46]: Not needed -> https://github.com/YunoHost/example_ynh/pull/171

Comment (fflorent) [reply scripts/restore:46]: I see, thanks! :+1:

**Pull request merged**